files/eslint.config.mjs

```diff
@@ -133,7 +121,4 @@ const config = [
       },
     },
   },
-];
-<% if (typescript) { %>
-export default ts.config(...config);<% } else { %>
```

Comment:
`ts.config` is deprecated in favor of `defineConfig`. I've verified locally, that the exact same configs are printed before and after.